junifer/external/nilearn/junifer_nifti_spheres_masker.py

```diff
@@ -104,7 +104,7 @@ def _apply_mask_and_get_affinity(
 
     # Compute world coordinates of all in-mask voxels.
     if niimg is None:
-        mask, affine = masking._load_mask_img(mask_img)
+        mask, affine = masking.load_mask_img(mask_img)
```

Comment:
so thanks to this we stop supporting nilearn 0.9?

Reply:
Yeah till 0.12 because of this and a few others.